packages/core/src/Stores/gtm-store.js

```diff
@@ -11,6 +11,8 @@ import { getMT5AccountType } from './Helpers/client';
 export default class GTMStore extends BaseStore {
     is_gtm_applicable = /^(16303|16929|19111|19112)$/.test(getAppId());
 
+    @observable is_tracking = false;
```

Comment:
Can we remove the changes made to this file regarding this observable, since it's not really needed as we only rely on the cookie anyway?

We only use this observable in the `AppContents` component file, however, we can just use internal state there. We should keep state as local as possible.